org.eclipse.lsp4e.test/pom.xml

```diff
@@ -20,6 +20,13 @@
 				<groupId>org.eclipse.tycho</groupId>
 				<artifactId>target-platform-configuration</artifactId>
 				<version>${tycho-version}</version>
+				<dependencies>
+					<dependency>
+						<groupId>org.eclipse.osgi</groupId>
+						<artifactId>org.eclipse.osgi.services</artifactId>
+						<version>3.2.100.v20100503</version>
+					</dependency>
+				</dependencies>
```

Comment:
This is not where dependencies must be added for efficient plugin development. The dependency set is defined in the .target file. Please add the required artifact to this .target file.

Reply:
I agree. It seems that this artifact is no longer provided by the p2 repos referenced in our target file. Could you find a p2 repo that has the artifact?

Reply:
Will do.